### PR TITLE
new file from AS0706

### PR DESCRIPTION
**Pull request by AS0707**
new file added

### CODE REVIEW

File: _posts/New1FromAS.md [hunk 1]
```diff
@@ -0,0 +1 @@
+;lfaksjdl;fa
```
Comment (MN0707) [inline _posts/New1FromAS.md:1]: I would add much more content

Comment (AS0707) [reply _posts/New1FromAS.md:1]: Ok, I'll do that ia a moment.

Comment (AS0707) [reply _posts/New1FromAS.md]: Done!

**Comment by MN0707** [comment]
Super! I'll merge to master branch! Cheers mate!

**Pull request merged**